test/integration/error.test.js

```diff
@@ -19,17 +20,19 @@ describe(Support.getTestDialectTeaser('Sequelize Errors'), function () {
       expect(sequelize).to.have.property('Error');
       expect(sequelize).to.have.property('ValidationError');
     });
-    it('Sequelize Errors instances should be instances of Error', function() {
-      var error = new Sequelize.Error();
-      var validationError = new Sequelize.ValidationError('Validation Error', [
-        new errors.ValidationErrorItem('<field name> cannot be null', 'notNull Violation', '<field name>', null)
-      , new errors.ValidationErrorItem('<field name> cannot be an array or an object', 'string violation', '<field name>', null)
-      ]);
-
 
-      var sequelize = new Sequelize();
-      var instError = new sequelize.Error();
-      var instValidationError = new sequelize.ValidationError();
+    it('Sequelize Errors instances should be instances of Error', function() {
+      var
+        error = new Sequelize.Error(),
+        errorMessage = "Validation Error",
+        validationError = new Sequelize.ValidationError(errorMessage, [
+          new errors.ValidationErrorItem('<field name> cannot be null', 'notNull Violation', '<field name>', null),
+          new errors.ValidationErrorItem('<field name> cannot be an array or an object', 'string violation', '<field name>', null)
+        ]),
+
+        sequelize = new Sequelize(),
+        instError = new sequelize.Error(),
+        instValidationError = new sequelize.ValidationError();
```

Comment:
Was anything actually changed here or is it pure comma styling?
If so please revert to not dirty the diff (we use leading commas in any case if it were to be changed :)